master/Makefile

```diff
@@ -109,6 +109,20 @@ check: check-gen
 	git diff --quiet go.mod go.sum
 	golangci-lint --build-tags integration run -v --timeout 10m
 
+.PHONY: fmt-sql
+fmt-sql:
+	# there are complaints from this tool that cannot be auto-fixed
+	# and we ignore them for now.
+	sqlfluff format -p 0 --dialect postgres . || exit 0
+	# sqlfluff fix -p 0 --dialect postgres . || exit 0
+
+.PHONY: check-sql
+check-sql:
```

Comment:
this is many times slower than other `check` tests we have in the `master` modules so I didn't add it to `make check`